### PR TITLE
Quick contribution count

### PR DESCRIPTION
**Pull request by joshuacrowley**
Show that designers are taking up the challenge and help motivate another PR contribution.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -84,7 +85,7 @@
             more.
         </p>
 
-        <p>Interested? Send a PR with any improvement to
+        <p>Interested? Join the <a href="https://github.com/Microsoft/join-dev-design/graphs/contributors"> <span id="contributors">3</span> contributors</a> that have sent a PR with an improvement to
```
Comment (dvdsgl) [inline docs/index.html]: I'd prefer for this not to be another link, but otherwise this is cool!

Comment (dvdsgl) [reply docs/index.html]: > Join the *n* contributors *who* have ...

**Comment by dvdsgl** [comment]
I love this but if you want me to merge it, could you perhaps add this is a more subtle tooltip on the `Open source on GitHub` element? I don't want to add this to the main message because it makes the call to action a bit less clear.

Don't worry about making the change though if it's too much trouble! Your PR has been noted!

Thank you.

**Comment by dvdsgl** [comment]
Now that we've already had 10 contributors, I would love to get this functionality in somewhere subtle.

I would accept `Developed by 10 contributors on GitHub.` in the footer.

**Comment by joshuacrowley** [comment]
@dvdsgl sounds good, I've made the change.

**Comment by dvdsgl** [comment]
The merge looks a bit messed up. Would you mind rebasing or just applying your changes to a fresh branch off master?

**Comment by joshuacrowley** [comment]
oops, looks good to me now. what do you think @dvdsgl?

**Comment by dvdsgl** [comment]
Strange, the PR still contains many unrelated commits. Perhaps open a new one?

Also, if you'd like, I can manually add your changes and list you as the author on the commit.

**Comment by joshuacrowley** [comment]
@dvdsgl feel free to manually add it, not fussed about attribution. I need to study up on my Git it seems. Thanks for the fun project though, I enjoyed contributing.

**Comment by dvdsgl** [comment]
Thank you!

After all these years I also sometimes find myself at the bottom of the git rabbit hole. I usually just copy files into a new clone at that point... Thanks again for your neat idea.

**Comment by dvdsgl** [comment]
Implemented in 796e3458128b821c992b0c8b43e3456aa71a4c70

**Pull request closed without merging**